consensus/bor/bor.go

```diff
@@ -41,6 +41,8 @@ import (
 )
 
 const (
+	spanLength         = 6400 // Number of blocks in a span
```

Comment:
So there is now an assumption that the span length is going to be fixed? Previously, I had an impression that span length could vary and is determined by something in heimdall, is it changed now?

Reply:
No, span length is fixed in heimdall and has been 6400 from beginning. I think you must have read about checkpoints. They're of dynamic length.

Reply:
Ok, in that case the code for getting spans does indeed look overly complicated. One can simply calculate the span number from the block number, right?

Reply:
Yes, I think we can skip the whole loop itself. Will push the updated code.

Reply:
Done. @AlexeyAkhunov can you review the changes once? Thanks!